dnf/logging.py

```diff
@@ -95,6 +96,21 @@ def _cfg_err_val2level(cfg_errval):
     return _ERR_VAL_MAPPING.get(cfg_errval, logging.WARNING)
 
 
+def compression_namer(name):
+    return name + ".gz"
+
+
+def compression_rotator(source, dest):
+    with open(source, "rb") as sf:
+        with gzip.open(dest, 'wb') as wf:
+            while True:
+                data = sf.read(1024)
```

Comment:
Okay, sorry, I should maybe have been more explicit the first time. 1024 is too small to be efficient, a quick google query found this:
https://eklitzke.org/efficient-file-copying-on-linux

128kB looks good.

Reply:
Thanks for the link! I changed the number.